docs(react-grid): data validation guide (#2685)
DevExpress/devextreme-reactive: packages/dx-react-grid/docs/guides/editing.md

> You can prevent editing of a specific column using the [EditingState](../reference/editing-state.md) plugin's `columnExtensions` property.

Reviewer: ```suggestion
You can use the [EditingState](../reference/editing-state.md) plugin's `columnExtensions` property to prevent a specific column for being edited.
```

.embedded-demo({ "path": "grid-editing/disable-column-editing", "showThemeSelector": true })

## Data Validation

> Changes can be validated on-the-fly as shown in the example below. The 'Save' button gets disabled if any reqired field is empty.

Reviewer 2: ```suggestion
Changes can be validated as the user types. You can disable the Save button if any required field is empty.
```

.embedded-demo({ "path": "grid-editing/validation-on-typing", "showThemeSelector": true })

> Another option is to validate data in the `onCommit` function. In this case it is possible to rollback invalid changes by omitting the `setState` call or display the error in a grid as in the demo below.

Reviewer 2: ```suggestion
Changes can also be validated in the `onCommit` function. In this case, you can omit the `setState` call and rollback the changes if they are invalid.
```
